cmake/multilib.yaml.in

```diff
@@ -174,6 +174,6 @@ Mappings:
   - -march=thumbv8.1m.main+lob
 
 # -mbranch-protection options
-- Match: -mbranch-protection=(standard|pac-ret(\+leaf)?\+bti|bti\+pac-ret(\+leaf)?)
+- Match: -mbranch-protection=(standard|pac-ret(\+leaf)?(\+bti)?|bti(\+pac-ret(\+leaf)?)?)
   Flags:
-  - -mbranch-protection=pac-ret+bti
+  - -mbranch-protection=standard
```

Comment:
I'm not sure this is the best approach to ensure the flags match. The `pac-ret+bti` value is more explicit in what it covers and less likely to suffer changes in meaning in the future if new branch protection features are introduced.
I feel that updating the expected flag value in the library variants to `pac-ret+bti` is probably a more stable approach.

Reply:
Thanks. Fixed.